operator/apis/platform/v1alpha1/securedcluster_types.go

```diff
@@ -190,17 +190,15 @@ type PerNodeSpec struct {
 	TaintToleration *TaintTolerationPolicy `json:"taintToleration,omitempty"`
 }
 
-// CollectionMethod defines the method of collection used by collector. Options are 'EBPF', 'CORE_BPF', 'KernelModule' or 'None'.
-// +kubebuilder:validation:Enum=EBPF;CORE_BPF;KernelModule;NoCollection
+// CollectionMethod defines the method of collection used by collector. Options are 'EBPF', 'CORE_BPF' or 'None'.
+// +kubebuilder:validation:Enum=EBPF;CORE_BPF;NoCollection
 type CollectionMethod string
 
 const (
 	// CollectionEBPF means: use EBPF collection.
 	CollectionEBPF CollectionMethod = "EBPF"
 	// CollectionCOREBPF means: use CORE_BPF collection.
 	CollectionCOREBPF CollectionMethod = "CORE_BPF"
-	// CollectionKernelModule means: use KERNEL_MODULE collection.
-	CollectionKernelModule CollectionMethod = "KernelModule"
```

Comment:
What happens if a customer still has "KernelModule" configured?
I am concerned that we introduce a backwards incompatible change which breaks customer instances.

Reply:
See my response to Aleksandr Kurlov.